.github/workflows/test.yaml

```diff
@@ -6,6 +6,20 @@ permissions:
   actions: read
 
 jobs:
+  clang-format:
+    # To pick up a newer clang
+    runs-on: ubuntu-24.04
```

Comment:
Any reason not to move the main workflow over to 24.04?  We're probably gonna get exposed to that in the next couple of months anyway (wieh it becomes the new `ubuntu-latest`)... makes sense to shake out any issues ahead of time.

Reply:
I added a comment about that https://github.com/containers/composefs/pull/326#issuecomment-2321626211

Reply:
Sorry.  Missed that.